### PR TITLE
Create Blog “redfish-interoperability-profiles”

### PR DESCRIPTION
**Pull request by donzef**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/redfish-interoperability-profiles.md
Comment (choukden) [inline content/blog/redfish-interoperability-profiles.md]: The code snippet is not rendered correctly. Please check the language syntax

File: content/blog/redfish-interoperability-profiles.md
Comment (choukden) [inline content/blog/redfish-interoperability-profiles.md:181]: Code snippet not correct

File: content/blog/redfish-interoperability-profiles.md
Comment (choukden) [inline content/blog/redfish-interoperability-profiles.md]: Code snippet not correct

**Review by choukden — CHANGES_REQUESTED** [review]
Bonjour Francois

Thank you for your new contribution.
I have added a few comments
Images and some code snippets are not correctly rendered.

The preview link is:
https://deploy-preview-2607--hpe-dev-portal.netlify.app/blog/redfish-interoperability-profiles/

I have asked Fredo or Mathieu to go through the Technical review.
Dale will go through the editorial review.

THANKS!!

Denis

File: content/blog/redfish-interoperability-profiles.md
Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:54]: Consider replacing to be started or stopped from remote. with to be powered on and off remotely

File: content/blog/redfish-interoperability-profiles.md
Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:17]: looking at me with big eyes and this question in mind:  to be replaced by looking at me puzzled and asking themselves...

File: content/blog/redfish-interoperability-profiles.md
Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:48]: can be a problem for  to be replaced by can turn into or become an issue

Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:48]: to move **away** from the Intelligent Platform Management Interface (<a href="https://www.intel.com/content/www/us/en/products/docs/servers/ipmi/ipmi-home.html" target="_blank">IPMI</a>) to Redfish for hardware management.

File: content/blog/redfish-interoperability-profiles.md
Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:50]: projects can only adopt Redfish as their preferred management technique   :  protocol innstead of technique ?? or technology

Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:50]: could be implemented in or on their managed nodes???

Comment (pattdepanick) [reply content/blog/redfish-interoperability-profiles.md:50]: Dale would know better ;-)

File: content/blog/redfish-interoperability-profiles.md
Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:54]: Consider replacing Systems eligible by the OpenStack  with Systems supported by

File: content/blog/redfish-interoperability-profiles.md
Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:56]: system cannot return the firmware version of its BMC  to be replaced by system cannot return its BMC's firmware version

File: content/blog/redfish-interoperability-profiles.md
Comment (pattdepanick) [inline content/blog/redfish-interoperability-profiles.md:65]: It is not relevant for use in a real life context. to replaced by  It is not relevant for use in a proper production context.

**Comment by donzef** [comment]
Reported issues hopefully resolved. Thanks. Waiting for Dale's review. No rush.

File: content/blog/redfish-interoperability-profiles.md
Comment (drensing) [inline content/blog/redfish-interoperability-profiles.md:235]: Recommend rewording this first sentence to:
Interoperability profiles can also be leveraged by purchasing departments of large organizations who have several computer suppliers. 

In the next sentence, change "could" to "can" (it's a more powerful word):
By including such profiles in their Request for Proposal (RFPs) documents, they can make sure that the computers they purchase...

The last sentence also needs some slight modification:
Having a clear knowledge of this baseline is **a** definite plus for security and supervising departments.

File: content/blog/redfish-interoperability-profiles.md
Comment (donzef) [reply content/blog/redfish-interoperability-profiles.md:58]: For now, I'll remove leading upper cases and ask for confirmation.

File: content/blog/redfish-interoperability-profiles.md
Comment (donzef) [reply content/blog/redfish-interoperability-profiles.md:174]: Aligned Interoperability Validator with Profiles: no upper case.

**Comment by donzef** [comment]
Implemented Dale's suggestions. Thanks.

Still need to rewrite a bit the last paragraph (as per feedback received offline) and get technical review from Jeff Autor.

**Comment by drensing** [comment]
[like]  Rensing, Dale reacted to your message:
________________________________

**Comment by drensing** [comment]
[like]  Rensing, Dale reacted to your message:
________________________________

**Pull request merged**
